inbm/dispatcher-agent/dispatcher/dispatcher_class.py

```diff
@@ -852,7 +853,7 @@ def invoke_sota(self, **kwargs) -> None:
         """
         logger.debug('Invoking SOTA')
 
-        parsed_manifest = {'sota_cmd': 'rollback', 'log_to_file': None,
+        parsed_manifest = {'sota_mode': self.sota_mode, 'sota_cmd': 'rollback', 'log_to_file': None,
                            'sota_repos': self.sota_repos,
```

Comment:
The switching back and forth between the turns mode and command is confusing for the same thing.  Since we are using command for the rollback, we probably should use mode throughout even if it's in the manifest with the 'cmd' tag.